agent/functional_tests/testdata/taskdefinitions/telemetry/task-definition.json

```diff
@@ -0,0 +1,10 @@
+{
+    "family": "ecs-metrics-test",
+    "containerDefinitions": [{
+        "memory": 100,
+        "name": "telemetry",
+        "cpu": 100,
+        "image": "ubuntu:latest",
```

Comment:
@aaithal thanks for pointing this out to me!

@RichardPen, I think the pull time for `ubuntu:latest` on a host with no cached images is contributing to the test timeout.  You'll want to do the same thing that we've done for our other test images.

Here's an example [task definition](https://github.com/aws/amazon-ecs-agent/blob/v1.8.1/agent/functional_tests/testdata/taskdefinitions/datavolume/task-definition.json#L18) with the image set to a registry running on `localhost` (started with our [`setup-test-registry`](https://github.com/aws/amazon-ecs-agent/blob/v1.8.1/scripts/setup-test-registry) script).  You'll also need to modify the script to add the `ubuntu` image [like this](https://github.com/aws/amazon-ecs-agent/blob/v1.8.1/scripts/setup-test-registry#L71-L77).